components/mdns/include/mdns.h

```diff
@@ -128,6 +128,18 @@ void mdns_free(void);
  */
 esp_err_t mdns_hostname_set(const char *hostname);
 
+/**
+ * @brief Get the hostname for mDNS server
+ *
+ * @param hostname      pointer to the hostname
```

Comment:
Could probably say that this `pointer to the hostname` needs to be allocated and hold at least `MDNS_NAME_BUF_LEN` chars.

Reply:
Added.